MessageBrokerPublisher.Tests/RegisterNotification.cs

```diff
@@ -0,0 +1,54 @@
+using MessageBrokerPublisher;
+using MessageBrokerPublisher.Models;
+using MessageBrokerPublisher.Services;
+using Moq;
+using NUnit.Framework;
+using RabbitMQ.Client;
+using System;
+using System.Text;
+
+namespace MessageBrokerPublisher.Tests
+{
+    public class Tests
+    {
+        
+        [Test]
+        public void RegisterNotification_EmailNotification_NotificationRegistered()
+        {
+            //arrange
```

Comment:
According to our wiki: `Begin comment text with an uppercase letter.` (https://github.com/DigitalExcellence/dex-backend/wiki/Coding-guidelines#commenting-conventions), I would suggest to follow this guideline.